homeassistant/components/roborock/__init__.py

```diff
@@ -99,6 +198,7 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
                 for coordinator in hass.data[DOMAIN][entry.entry_id].values()
             )
         )
+        await Store(hass, STORE_VERSION, DOMAIN).async_remove()
```

Comment:
It looks like the cached information does't need to survive a restart.
Therefore, please use `hass.data[DOMAIN]`, which is runtime only, instead of a store

Reply:
This line shouldn't have been here - removed